spec/factories/users.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :user do
```

Comment:
define the attributes for your factory so you don't have to define them in each test